### PR TITLE
Yo, this whitespace will trigger noobs

### PR DESCRIPTION
**Pull request by JanTuck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ How To (Plugin Developers)
     <artifactId>paper-api</artifactId>
     <version>1.16.1-R0.1-SNAPSHOT</version>
     <scope>provided</scope>
- </dependency>
+</dependency>
```
Comment (JRoy) [inline README.md:41]:
```suggestion
 </dependency>
```

**Review by JRoy — CHANGES_REQUESTED** [review]

**Review by pop4959 — CHANGES_REQUESTED** [review]
It's not a bug, it's a feature

**Review by mdcfe — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JanTuck) [reply README.md:41]: Triggered

Comment (Chew) [reply README.md:41]: > Triggered

Does this mean you're a noob?

Comment (JanTuck) [reply README.md:41]: Maybe,

**Pull request merged**